poolapi/apijson.go

```diff
@@ -0,0 +1,41 @@
+package poolapi
+
+type Response struct {
+	Status  string      `json:"status"`
+	Message string      `json:"message"`
+	Data    interface{} `json:"data,omitempty"`
+}
+
+func NewResponse(status, message string, data interface{}) *Response {
+	return &Response{status, message, data}
+}
+
+type PurchaseInfo struct {
+	PoolAddress   string `json:"pool_address"`
```

Comment:
Right, I did this in several spots as it seemed more idiomatic json, but I'm not strongly in favor of this.